src/pages/organize/[orgId]/projects/[campId]/events/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { GetServerSideProps } from 'next';
+import { scaffold } from 'utils/next';
+
+export const getServerSideProps: GetServerSideProps = scaffold(async () => {
+  return {
+    redirect: {
+      destination: '../calendar',
+      permanent: false,
+    },
+  };
+});
```

Comment:
Interpolating an absolute path should work, something like this (from memory, so might need modifications):

```suggestion
export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
  const { orgId, campId } = ctx.params!;
  return {
    redirect: {
      destination: `/organize/${orgId}/projects/${campId}/calendar`,
      permanent: false,
    },
  };
});
```